### PR TITLE
Adding Fossfox

### PR DESCRIPTION
**Pull request by 4kimov**

### CODE REVIEW

**Comment by tramcarjobs** [comment]
@4kimov thanks for the PR (and apologies for the slow update!)

I have a quick question about this board before I merge. Is this a job board where jobs will allow you to work _with_ open source technologies or are they jobs where you'll be working _on_ open source technologies? The description makes it sound like the latter, but the jobs themselves look more like the former. Can you please clarify?

**Comment by 4kimov** [comment]
@tramcarjobs Thanks for following up!

To answer your question: It's a job board where the companies you work for have their main product open-sourced (or at least are very heavy contributors to complimentary products). The jobs themselves can be either directly for that open-sourced product, or they can be in support of (open-source companies still need someone to handle devops or website landing pages).

Basically, it lists opportunities to work with companies that embrace open-source.

**Comment by tramcarjobs** [comment]
@4kimov perfect, thanks for clarifying. Could you please update the note ("Opportunity explorer for open-source devs") to better reflect the "opportunities to work with companies that embrace open-source" nature?  Once updated, I'd be happy to merge this. TY!

**Comment by 4kimov** [comment]
@tramcarjobs No problem, adjusted. Is this better?

**Comment by tramcarjobs** [comment]
LGTM, thanks for making that change @4kimov !

**Comment by tramcarjobs** [comment]
@4kimov hey there, I've removed this site due to the following error:

```
5. [L201]  https://fossfox.com/ Failed to open TCP connection to fossfox.com:443 (getaddrinfo: No address associated with hostname) 
```

Feel free to add back if/when it's fixed.
